src/mds/MDSRank.cc

```diff
@@ -326,6 +325,9 @@ void MDSRankDispatcher::tick()
   if (is_clientreplay() || is_active() || is_stopping()) {
     server->find_idle_sessions();
     locker->tick();
+    if (is_active()) {
+      server->evict_cap_revoke_non_responders();
```

Comment:
eviction may also make sense in `up:stopping`. @ukernel what do you think?

Reply:
I think that should be fine -- `Server::find_idle_sessions()` called above can evict clients too. I think it should be ok to just invoke `Server::evict_cap_revoke_non_responders()` here (without any additional check). @ukernel?

Reply:
yes